pyproject.toml

```diff
@@ -62,6 +62,3 @@ junit_family = "xunit2"
 markers = """
     conf_file:Configuration file.
 """
-
-[tool.black]
-exclude = "(dist/)|(build/)"
```

Comment:
We don't want to exclude these?

Reply:
Back was replaced by Ruff formatter in #1264 and here just removing missed artifact